From 808219a79843a66190c44105575623be40f74304 Mon Sep 17 00:00:00 2001
From: Tommy Huang <tommy_huang@aspeedtech.com>
Date: Tue, 9 Sep 2025 19:23:12 +0800
Subject: [PATCH 03/10] dts: ast2700: add timeout setting for mailbox 2

Add tx-timeout setting for mailbox 2.

Signed-off-by: Tommy Huang <tommy_huang@aspeedtech.com>
Change-Id: If53780df51f35a6625f78c4411004d8128b7e16b
(cherry picked from commit 292f9e0879dab54f13bc4f66627e8632004b3c52)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/arm/dts/ast2700.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/ast2700.dtsi b/arch/arm/dts/ast2700.dtsi
index 070037d6e7..83c2c875ec 100644
--- a/arch/arm/dts/ast2700.dtsi
+++ b/arch/arm/dts/ast2700.dtsi
@@ -965,6 +965,7 @@
 			reg-names = "tx", "rx";
 			#mbox-cells = <1>;
 			mbox-name = "bootmcu";
+			aspeed,tx-timeout = <1000>;
 		};
 	};
 };
-- 
2.34.1

